### PR TITLE
DX-1288 Webrtc 0.1.0

### PR DESCRIPTION
**Pull request by jturner-bw**
First round of WebRTC docs. These were auto generated, so please let me know how the format fits and I'm happy to share the code

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

File: webrtc/errors.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Errors
```
Comment (jmulford-bw) [inline webrtc/errors.md:1]: I think this was copied over from MFA. Feel free to update it with some WebRTC content, or just remove it if it's not necessary right now

**Review by jmulford-bw — CHANGES_REQUESTED** [review]
I think we already discussed this but once the response body examples are filled in this is 👍 Awesome job on the autogeneration of this

File: webrtc/errors.md (hunk 1 above)
Comment (jturner-bw) [reply webrtc/errors.md:1]: yep good catch. just updated

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — APPROVED** [review]
I'm guessing there's no error response body? No worries for now if that's the case 👍

**Comment by jturner-bw** [comment]
> I'm guessing there's no error response body? No worries for now if that's the case 👍

Not sure we have those totally worked out yet, but yes we will/should have them. If you think we're ok for now, I'd like to get this live and we can come back and clean that up soon

**Comment by jmulford-bw** [comment]
> > I'm guessing there's no error response body? No worries for now if that's the case 👍
> 
> Not sure we have those totally worked out yet, but yes we will/should have them. If you think we're ok for now, I'd like to get this live and we can come back and clean that up soon

Fine with me. I'll get Dan to look at this in case he sees something out of the ordinary but I think this should be good

**Comment by dtolb** [comment]
The HTTP responses aren't pulling in formatting

image

Here is how to format correctly:

image

---

Finally, in the method examples, we _traditionally_ don't put the errors there; that said: I do like how the information is displayed. Perhaps on the `errors` page put examples on how you could come across such errors [(messaging example)](http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/messaging/errors/httpErrors.html#top)

However, in the pages; I'd like to mark those as "possible" errors within that page.

an H3 `### Potential Error Responses` should sove it.

image

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

File: webrtc/methods/about.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| <code class="delete">DELETE</code>              | [`/sessions`](/sessions/deleteSession.md)                                                         | Delete session by ID                                                                              
+| <code class="put">PUT</code>                    | [`/sessions`](/sessions/addParticipantToSession.md)                                               | Add a participant to a session                                                                    
+| <code class="delete">DELETE</code>              | [`/sessions`](/sessions/removeParticipantFromSession.md)                                          | Remove a participant from a session                                                               
+| <code class="get">GET</code>                    | [`/sessions`](/sessions/getParticipantSubscriptions.md)                                           | Get a participant's subscriptions                                                                 
+| <code class="put">PUT</code>                    | [`/sessions`](/sessions/updateParticipantSubscriptions.md)                                        | Update a participant's subscriptions
```
Comment (dtolb) [inline webrtc/methods/about.md]: These links aren't resolving

example

Comment (jturner-bw) [reply webrtc/methods/about.md]: 🤦 fixed

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by jturner-bw** [comment]
@dtolb thanks for feedback, unless I hear otherwise from you or @jmulford-bw I'm assuming this is good to merge

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-webrtc-0.1.0.s3-website-us-east-1.amazonaws.com/

**Pull request merged**